src/snowflake/cli/app/snow_connector.py

```diff
@@ -26,11 +25,10 @@
 def connect_to_snowflake(temporary_connection: bool = False, connection_name: Optional[str] = None, **overrides) -> SnowflakeConnection:  # type: ignore
 
     if not temporary_connection:
-        connection_parameters = (
-            get_connection(connection_name)
-            if connection_name is not None
-            else _get_default_connection_params()
-        )
+        if connection_name is not None:
+            connection_parameters = get_connection(connection_name)
```

Comment:
It might be a little confusing, hence `get_connection` does not return connection nor session but connection parameters.